easybuild/easyconfigs/f/FIAT/FIAT-1.6.0-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
+
+name = 'FIAT'
+version = '1.6.0'
+
+homepage = 'https://bitbucket.org/fenics-project/fiat'
+description = """The FInite element Automatic Tabulator FIAT supports generation of arbitrary order
+ instances of the Lagrange elements on lines, triangles, and tetrahedra. It is also capable of generating
+ arbitrary order instances of Jacobi-type quadrature rules on the same element shapes."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://bitbucket.org/fenics-project/fiat/downloads']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('sympy', '1.0', '-Python-%(pyver)s'),
```

Comment:
@RvDijk name of easyconfig file doesn't match (currently empty) `versionsuffix`, so please change to:

```python
version = '1.6.0'
versionsuffix = '-Python-%(pyver)s'
...
    ('sympy', '1.0', versionsuffix),
```